crates/bevy_reflect/bevy_reflect_derive/src/impls/enums.rs

```diff
@@ -192,8 +192,8 @@ pub(crate) fn impl_enum(reflect_enum: &ReflectEnum) -> TokenStream {
             }
 
             #[inline]
-            fn get_type_info(&self) -> &'static #bevy_reflect_path::TypeInfo {
-                <Self as #bevy_reflect_path::Typed>::type_info()
+            fn represented_type_info(&self) -> #FQOption<&'static #bevy_reflect_path::TypeInfo> {
```

Comment:
nit: wondering if this should be `get_represented_type_info`, in line with other methods returning `Option<T>`.

Reply:
Great call out. I'll change that!